src/pybind/mgr/tox.ini

```diff
@@ -149,14 +150,14 @@ commands =
     autopep8 {[autopep8]addopts} \
       {posargs:{[testenv:fix]modules}}
 
-[testenv:pylint]
+[testenv:lint]
 basepython = python3
 deps =
     pylint
 modules =
     cli_api
 commands =
-    pylint {[pylint]addopts} {posargs:{[testenv:pylint]modules}}
+    pylint {[pylint]addopts} {posargs:{[testenv:lint]modules}}
 
```

Comment:
what I don't understand is: this tox env doesn't exist in master.

Reply:
> what I don't understand is: this tox env doesn't exist in master.

@sebastian-philipp This branch PR is against a feature branch (`feature-48388-cache`), not master.
The linting would be run against this module only for now, so no worries about lint errors in other modules.